demos/wascc/fileserver/Makefile

```diff
@@ -0,0 +1,37 @@
+COLOR ?= always # Valid COLOR options: {always, auto, never}
```

Comment:
Any reason you used `make` over `just` here?

Reply:
copypasta from the `uppercase` demo.

Reply:
Nah, no hard requirement here